tabs/src/components/app/views/RSCList.jsx

```diff
@@ -27,14 +28,18 @@ export function RSCList() {
 
             if (context) {
                 const rscResponse = await makeGraphCall(requestTypes.GET, [], rscType + "/permissionGrants", graphVersions.beta);
+
                 if (!rscResponse.ok) {
                     setAlert(true);
+                    setIsLoading(false);
```

Comment:
Don't quite understand why loading is set to false only when `!rscResopnse.ok`..

Reply:
There is a second usage for it

Reply:
Line 62